docs/sql-blocks.md

```
@@ -0,0 +1,366 @@
+   # MySQL
+   pip install pymysql
+
+   # Snowflake
+   pip install snowflake-connector-python
+
+   # BigQuery
+   pip install google-cloud-bigquery
+   ```
+
+3. **Environment variables** set with connection details:
+   ```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)